src/test/java/cz/czechitas/Page.java

```diff
@@ -0,0 +1,31 @@
+package cz.czechitas;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.*;
+
+public class Page {
+    WebDriver driver;
+
+    public Page(WebDriver driver){
+        this.driver = driver;
+    }
+
+    public void login(String loginPassword, String loginUserName){
+        WebElement user = driver.findElement(By.id("username"));
+        WebElement password = driver.findElement(By.id("password"));
+        WebElement button = driver.findElement(By.id("formSubmitButton"));
+        user.sendKeys(loginUserName);
+        password.sendKeys(loginPassword);
+        button.click();
+    }
+
+    public Boolean getRightAddress(String urlAddress){
+        Boolean isAddressRight = new WebDriverWait(driver, 5).until(d->d.getCurrentUrl().equals(urlAddress));
+        return isAddressRight;
+    }
+
+    public WebElement getElementByClassName(String className){
```

Comment:
Tohle nema to zadnou pridanou hodnotu nad prime pouziti slenia. Kdyz us delas PO, je nesmysl vracet z jeho metod WebElement. Spravne by bylo neco jako:
```
public String getErrorMessage() {
    WebElement element = new WebDriverWait(driver, 5).until(d->d.findElement(By.className('alert-error')));
    return element.getText();
}
```